applications/luci-app-acme/htdocs/luci-static/resources/view/acme.js

```diff
@@ -579,6 +585,15 @@ return view.extend({
 	}
 })
 
+function _isFqdn(domain) {
+	// Is not an IP i.e. starts from alphanumeric and has least one dot
+	return /[a-z0-9-]\..*$/.test(domain);
```

Comment:
I don't think this regex does what you want it to. It will match, for instance, '1.'.

Maybe something like:
`return (! /^[0-9\.]*$/.test(domain)) && /\./.test(domain);`

I.e., something that has at least one dot in it, but is not made exclusively of numbers and dots?

Reply:
I understand what you're trying to do, I'm just saying that the regex you wrote does not accomplish this task. `/[a-z0-9-]\..*/` will match any numerical IP, for instance...

Reply:
This regex definitely does not do what you think it does. 
`[a-z0-9-]` matches these character ranges, but no quantifier follows it. So: 
a.xyz.com ?

Test on https://regexr.com/

There is already a hostname datatype test.

Reply:
Might be an idea to test for Ipv4 and 6, and if not those, assume it's an fqdn instead?